src/encoded/static/components/dataset.js

```diff
@@ -2270,6 +2271,84 @@ const generateSupplementalShortLabels = (experimentList, getSupplementalShortLab
 );
 
 
+/**
+ * Defines the number of experiments/control experiments that appear at once in the series dataset
+ * table.
+ */
+const PAGE_DATASET_COUNT = 25;
+
+
+/**
+ * Display one of the two sections (experiments or control experiments) of the dataset table on the
+ * Series pages.
+ */
+const SeriesDatasetTableSection = ({ title, datasets, tableColumns, sortColumn, adminUser, accessLevel }) => {
+    /** The current page of datasets if more than the max number of displayable datasets exist */
+    const [currentDatasetPage, setCurrentDatasetPage] = React.useState(0);
+
+    /**
+     * Called when the user clicks the pager to go to a new page of datasets.
+     * @param {number} New current page number of datasets to display
+     */
+    const updateCurrentExperimentPage = (newPage) => {
+        setCurrentDatasetPage(newPage);
+    };
+
+    const totalDatasetPageCount = Math.floor(datasets.length / PAGE_DATASET_COUNT) + (datasets.length % PAGE_DATASET_COUNT !== 0 ? 1 : 0);
```

Comment:
It may be a good idea to say you are adding one because Javascript start collections from 0 and the pager starts from 1, so you are trying to sync the two.